lib/cherry_pick.js

```diff
@@ -120,68 +100,6 @@ export default class CheckPick {
     }
   }
 
-  async downloadAndPatch(expectedCommitShas) {
-    const { cli, repo, owner, prid } = this;
-
-    cli.startSpinner(`Downloading patch for ${prid}`);
-    // fetch via ssh to handle private repo
-    await runAsync('git', [
-      'fetch', `git@github.com:${owner}/${repo}.git`,
-      `refs/pull/${prid}/merge`]);
-    // We fetched the commit that would result if we used `git merge`.
-    // ^1 and ^2 refer to the PR base and the PR head, respectively.
-    const [base, head] = await runAsync('git',
-      ['rev-parse', 'FETCH_HEAD^1', 'FETCH_HEAD^2'],
-      { captureStdout: 'lines' });
-    const commitShas = await runAsync('git',
-      ['rev-list', `${base}..${head}`],
-      { captureStdout: 'lines' });
-    cli.stopSpinner(`Fetched commits as ${shortSha(base)}..${shortSha(head)}`);
-    cli.separator();
-
-    const mismatchedCommits = [
-      ...commitShas.filter((sha) => !expectedCommitShas.includes(sha))
-        .map((sha) => `Unexpected commit ${sha}`),
-      ...expectedCommitShas.filter((sha) => !commitShas.includes(sha))
-        .map((sha) => `Missing commit ${sha}`)
-    ].join('\n');
-    if (mismatchedCommits.length > 0) {
-      throw new Error(`Mismatched commits:\n${mismatchedCommits}`);
-    }
-
-    const commitInfo = { base, head, shas: commitShas };
-
-    try {
-      await forceRunAsync('git', ['cherry-pick', `${base}..${head}`], {
-        ignoreFailure: false
-      });
-    } catch (ex) {
-      await forceRunAsync('git', ['cherry-pick', '--abort']);
```

Comment:
`LandingSession` doesn't abort the cherry-pick on failure.

Reply:
Yeah, that's one of the inconsistency I'm trying to get rid of, the `--abort` call was removed in 1e6f5d3cbc7e837f1590122458b15f24ff9b378d, but not in this file.